spec/generators/rspec/system/system_generator_spec.rb

```diff
@@ -0,0 +1,47 @@
+# Generators are not automatically loaded by rails
+require 'generators/rspec/system/system_generator'
+require 'support/generators'
+
+RSpec.describe Rspec::Generators::SystemGenerator, :type => :generator do
+  setup_default_destination
+
+  describe "system specs" do
+    subject(:system_spec) { file("spec/system/posts_spec.rb") }
+    if ::Rails::VERSION::STRING < '5.1'
```

Comment:
Do we want this as a gate or do we want to tag the example to don't be run under Rails 5.1?